### PR TITLE
feat(menuLinks): add about page and links to menu

### LINKED ISSUE #402: Add standalone downloads
- [ ] Add download links for the three electron bundles to menu page
- [ ] links should start a download of the respective bundle
- [ ] list them under each other
- [ ] give them icons?
  - https://materialdesignicons.com/icon/microsoft-windows
  - https://materialdesignicons.com/icon/ubuntu
  - https://materialdesignicons.com/icon/apple

### LINKED ISSUE #423: Menu: fill in links
- [ ] make the 'About this project' page a mark down page similar to the about pages for the datasets
  - we should keep a list of all contributors there. :)
    - @ KatvonRivia 
    - @ pwambach 
    - @ natastro
    - @ pmast
- [ ] link the 'ESA Website' item to `https://www.esa.int/`
- [ ] link the 'CCI Website' item to `https://climate.esa.int/`
- [ ] link the github item to: `https://github.com/ubilabs/esa-climate-from-space`
- [ ] open external links in a new tab

### PR DESCRIPTION
**Pull request by KatvonRivia**
closes #423
closes #402

### CODE REVIEW

File: src/scripts/components/menu/menu.tsx [hunk 1]
```diff
@@ -8,67 +8,132 @@ import {LanguageIcon} from '../icons/language-icon';
...
+              label={'showcaseMode'}
+              link={'/showcase'}
+            />
+          </li>
+          <li className={styles.subMenuTitle}>
+            <DownloadIcon /> <FormattedMessage id={'offline'} />
+          </li>
+          <li className={styles.menuListItem}>
+            <a
+              href={`https://storage.googleapis.com/esa-cfs-versions/electron/${version}/esa-climate-from-space-${version}-win.exe`}
```
Comment (pwambach) [inline src/scripts/components/menu/menu.tsx:50]: Would be great if the urls come from the main config file similar to the other urls like `baseUrlTiles` etc. You can insert the version there already.

File: src/scripts/components/about-project/about-project.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const AboutProject: FunctionComponent = () => (
+  <div className={styles.aboutProject}>
+    <div className={styles.title}>
+      <h1>
+        <FormattedMessage id="about" />
+      </h1>
+    </div>
+    <div className={styles.description}>
+      <FormattedMessage id="projectDescription" />
```
Comment (pwambach) [inline src/scripts/components/about-project/about-project.tsx:14]: The about page should be markdown right?

Comment (KatvonRivia) [reply src/scripts/components/about-project/about-project.tsx:14]: I thought we agreed on having it in the translations. 🤔

Comment (pwambach) [reply src/scripts/components/about-project/about-project.tsx:14]: Yes, but with markdown support :)

Comment (KatvonRivia) [reply src/scripts/components/about-project/about-project.tsx:14]: Ah, I see! Thanks, will add it.

**Pull request merged**